src/test/resources/features/account.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Accounts
+
+  Background:
+    Given I sign in as "owner"
+    And I go to accounts
+    And I create account "Account To Delete"
+
+  Scenario: Delete Account
+    Then I delete this account and notice "Account To Delete was successfully deleted." is displayed
+    When I go to accounts
+    Then I validate account "Account To Delete" is not displayed
+    When I go to dashboard
+    Then I verify account "Account To Delete" is not displayed in create project dropdown
```

Comment:
You should have only one Given, When - Then clauses followed the And instructions 

for example:
```gherkin
Background:
    Given I do the first precondition
    And I do the second precondition
    And I do the third precondition

  Scenario: Delete Account
    When I do the first step
    And I do the second step
    And I do the third step
    Then I do the first validation
    And I do the second validation
    And I do the third validation
```
Also it is missing the validation in the project setting form.